website/project/views/node.py

```diff
@@ -856,7 +856,7 @@ def private_link_table(node, **kwargs):
 @must_be_valid_project
 def get_editable_children(auth, node, **kwargs):
 
-    if not node.can_edit(auth):
+    if not node.has_permission(auth.user, 'admin'):
```

Comment:
Perhaps we temporarily add @must_have_permission(ADMIN) to the view for now. It will prevent users with write access from creating VOLs, but perhaps this is desired anyway? 

@jlcohoon What is the desired/expected permissions for creating VOLs?